src/Database/helpers.d.ts

```diff
@@ -0,0 +1,6 @@
+import { BatchOperationType } from "@nozbe/watermelondb/adapters/type";
+
+declare module '@nozbe/watermelondb/Database/helpers' {
```

Comment:
I could use some guidance her @radex - have never written ts before - is this correct?

Reply:
TBH I don't know :) We don't use TypeScript internally, 🍉's TS support is entirely community-led

Reply:
TBH “But why not TypeScript”? :D https://sobolevn.me/2019/03/from-flow-to-typescript

I hope to look at it this week.